deploy-sentinel/teletraan/RESTARTING

```diff
@@ -4,5 +4,9 @@ set -e
 
 echo Running RESTARTING script.
 
+# Register knox keys
+knox register -k teletraan:prod:rodimusapitoken
```

Comment:
`knox register -g -k <key>`  will also fetch it, `-k` only adds it to the local config file (fetches every 10 minutes)